go.mod

```diff
@@ -3,8 +3,9 @@ module github.com/terraform-providers/terraform-provider-aci
 go 1.12
 
 require (
-	github.com/ciscoecosystem/aci-go-client v1.7.14
+	github.com/ciscoecosystem/aci-go-client v1.9.0
 	github.com/ghodss/yaml v1.0.0
+	github.com/hashicorp/terraform v0.12.5 // indirect
```

Comment:
Do we still need that github.com/hashicorp/terraform v0.12.5 // indirect ??

Reply:
yes we do, it falls under basic requirements